x-pack/plugins/apm/server/lib/services/get_service.ts

```diff
@@ -28,15 +30,8 @@ export async function getService(
 
   const filter: ESFilter[] = [
     { term: { [SERVICE_NAME]: serviceName } },
-    {
-      range: {
-        '@timestamp': {
-          gte: start,
-          lte: end,
-          format: 'epoch_millis'
-        }
-      }
-    }
+    { terms: { [PROCESSOR_EVENT]: ['error', 'transaction'] } },
```

Comment:
Note: possible to specify several terms at once (thanks @ogupte for showing me this)